ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/BaseIntegrationTests.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  PurchasesHybridCommonIntegrationTests.swift
+//  PurchasesHybridCommonIntegrationTests
+//
+//  Created by Nacho Soto on 6/27/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+@testable import RevenueCat
+import XCTest
+
+class BaseIntegrationTests: XCTestCase {
```

Comment:
This is pretty much the same setup as in the iOS SDK.
Doesn't feel great to duplicate this code, but I don't think it's worth sharing this.